streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -106,10 +111,13 @@
     private final RocksDBMetricsRecorder metricsRecorder;
 
     protected volatile boolean open = false;
+    // VisibleForTesting
+    protected Position position;
+
     private StateStoreContext context;
-    private Position position;
 
-    RocksDBStore(final String name,
+    // VisibleForTesting
+    public RocksDBStore(final String name,
```

Comment:
This is needed so that I can test the consistency in the integration test. I need access to the `Position` so that I can assert it is correct

Reply:
Update: I didn't wind up returning the position information in the framework PR, so the comment still applies. I think we'll revise this once we're able to get the position from the public API, but for now this is just fine.